src/sonic-host-services-data/templates/common-auth-sonic.j2

```diff
@@ -23,10 +23,23 @@ auth	[success=1 default=ignore]	pam_tacplus.so server={{ last_server.ip }}:{{ la
 
 {% endif %}
 {% elif auth['login'] == 'tacacs+' or auth['login'] == 'tacacs+,local' %}
+{% if not auth['failthrough'] %}
+{% if auth['check_readonly_disk'] %}
```

Comment:
the feature is probably better named as failthrough_on_readonly_disk?

Reply:
ok